src/coreclr/src/jit/hwintrinsicarm64.cpp

```diff
@@ -337,6 +337,16 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             break;
         }
 
+        case NI_Vector64_get_Zero:
+        case NI_Vector128_get_Zero:
+        case NI_Vector64_get_AllBitsSet:
+        case NI_Vector128_get_AllBitsSet:
+        {
```

Comment:
You probably don't need this - if you change a category to SimpleSIMD the importation should be automatically handled as I said below